src/macros.rs

```diff
@@ -0,0 +1,17 @@
+#[macro_export]
+macro_rules! exit {
+    ($ec:expr, $($message:expr), +) => {
+        {eprintln!($($message), +);
+        std::process::exit($ec);}
+    };
+}
+
+#[macro_export]
+macro_rules! matches_any {
+    ($value:expr, $first:expr) => {
+        $value.eq_ignore_ascii_case($first)
```

Comment:
`eq_ignore_ascii_case` is from the unstable (in the latest stable & beta rust) `osstring_ascii` unfortunately for an unrelated problem I couldn't test the code with nightly either. If our current version dependency is that specific we should write it down somewhere.